README.md

```diff
@@ -33,6 +33,7 @@ Conduit was created and open-sourced by [Meroxa](https://meroxa.io).
 - [API](#api)
 - [UI](#ui)
 - [Documentation](#documentation)
+- [Configuration](#configuration)
```

Comment:
This is misplaced, it should be above `Documentation`.

Thinking out loud here - does this deserve its own top-level header? Especially since it's only 1 sentence right now, but also because this list is getting a bit crowded IMO. What if we add that sentence in the `Download release` chapter where we explain how to run Conduit? We can hint that they can check the help to see config options.

Reply:
Agreed. :+1: